components/language-select/__test__.js

```diff
@@ -0,0 +1,22 @@
+describe('Translations toggle successfully', () => {
+  it('Translates from EN to ZH CH', () => {
+    cy.intercept({
+      url: '/',
+    });
+    cy.visit('/');
+    cy.get('#downshift-0-toggle-button').click()
+      .invoke('text')
+      .then((text) => {
+        expect(text).to.equal('Select language');
+        cy.get('[data-code="zh_CN"]').click();
```

Comment:
I think we should test that some element, like any text out of this select, is translated properly from one language to another and vice-versa.